sklearn/impute/_iterative.py

```diff
@@ -697,11 +706,29 @@ def fit_transform(self, X, y=None):
         y : Ignored
             Not used, present for API consistency by convention.
 
+        **fit_params : dict
+            Parameters routed to the `fit` method of the sub-estimator via the
+            metadata routing API. Only available if
+            `sklearn.set_config(enable_metadata_routing=True)` is set.
+
+            See :ref:`Metadata Routing User Guide <metadata_routing>` for more
+            details.
+
+            .. versionadded:: 1.5
```

Comment:
This part is better as a note to the `versionadded` directive:

            Only available if
            `sklearn.set_config(enable_metadata_routing=True)` is set.

            See :ref:`Metadata Routing User Guide <metadata_routing>` for more
            details.